live-streaming/restreams.md

```diff
@@ -43,29 +43,31 @@ The Live stream endpoint has an optional field called `restreams` which is an ar
 | Field       | Type     | Details                                                                                           |
 | :---------- | :------- | :------------------------------------------------------------------------------------------------ |
 | `name`      | `string` | Arbitrary value for the name of the provider streaming provider. For example: `Youtube`, `Twitch` |
-| `serverUrl` | `string` | The RTMP url of the streaming server.                                                             |
+| `serverUrl` | `string` | The URL of the streaming server. The API accepts server URLs using `RTMPS` and `RTMP` protocols. |
 | `streamKey` | `string` | Stream key provided by the streaming provider.                                                    |
 
 <Callout pad="2" type="warning">
 Please note that:
 
 - Currently api.video supports up to 5 restreams
-- At this time, it's only possible to restream to RTMP
+- You can stream to `RTMPS` and `RTMP` destinations
 - The names of the providers are arbitrary, you can provide any name you like
 - When updating the restreams object, it's important to make sure that you are passing the previous values if you would like to keep them
 - Modifying the restreams array while the live stream is already broadcasting will only take only effect after the stream has been restarted
 </Callout>
 
 ## Supported Platforms
 
-api.video currently supports RTMP restreaming destinations. Providers that you can restream to include, and are not limited to:
+api.video currently supports `RTMPS` and `RTMP` restreaming destinations. Providers that you can restream to include, and are not limited to:
 
-- YouTube
+- Youtube
+- Facebook
+- Instagram
 - Twitch
 - LinkedIn
 - Instagram (through [YellowDuck](https://yellowduck.tv/))
 
-Providers that use RTMPS, SRT or any other protocol (like Kick or Telegram) are currently not supported.
+Providers that use SRT or any other protocol (like Kick or Telegram) are currently not supported.
```

Comment:
I don't know about the protocol supported by other platforms in details, it needs to be tested to test which works and which doesn't